docs/version-control.md

```diff
@@ -0,0 +1,45 @@
+# Version Control
+These standards are based on Defra Digital's [version control standards](https://github.com/DEFRA/software-development-standards/blob/master/standards/version_control_standards.md), but are extended to cover not only the application version, but version control of Docker images and Helm charts.
+
+## Semantic Versioning
+We use semantic versioning to manage code, images and charts.
+
+Given a version number MAJOR.MINOR.PATCH, increment the:
+- MAJOR version when you make incompatible API changes,
+- MINOR version when you add functionality in a backwards compatible manner, and
+- PATCH version when you make backwards compatible bug fixes.
+
+Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
+
+Full details can be found at [semver.org](semver.org).
```

Comment:
you are missing the `https://` before the URL, the link isn't working correctly in the rendered page

Reply:
Fixed